.github/workflows/package.yml

```diff
@@ -139,6 +157,12 @@ jobs:
       - name: Package
         run: cmake --build "$BUILD_DIR" --target package
 
+      - name: Generate artifact attestation
+        id: attestation
```

Comment:
For my culture why this job need an id ?

Reply:
To be honest, I don't know also. Their example has this structure so i just copied 😁

Reply:
I think you can drop it if you don't use a variable like ${{ steps.attestation.XXXX }}

Reply:
yes I agree 👍